### ISSUE TITLE
tag inputs should be space-delimited

### ISSUE THREAD

**Issue by FND** [innoq/iBlog#199]
tagging used to allow both space- and comma-delimited inputs (cf. db0b1a49e368327c4f9a0e01168a3ff276358d8c) - now it appears tags must be comma-delimited, quite possibly due to the switch to ActsAsTaggableOn (cf. 5ab1baa5d7be325b384583d3cfe00dfd9f142e68)

@mjansing: I assume we just need to add some processing to `tag_list` somewhere? I'm afraid it's too late in the day for me to delve into Rails magic myself...

**Comment by mjansing** [comment]
That's not a big deal. We have to add an initializer to set act_as_taggable-configuration like `ActsAsTaggableOn.delimiter = [',', ' ']` as described in https://github.com/mbleigh/acts-as-taggable-on#configuration.

But why you do not use the default delimiter (`,`)?

**Comment by mjansing** [comment]
I add an option to allow tags to be comma- and space-delimited.

**Comment by FND** [comment]
Thanks for the fix.

> But why you do not use the default delimiter (`,`)?

Well, this change wasn't documented anywhere, so a number of posts (not just my own) were mistagged. Indeed, some still are (e.g. @aknrdureegaesr's post from about a week ago) - so we'll also need a simple database migration to turn any space-delimited tag into a list of tags?

**Comment by mjansing** [comment]
Hmm that's true. I checked the database for space-delimited tags with [1] and I find six tags including the one you mentioned by @aknrdureegaesr. Only one of these tags is referenced [2] to an entry (also the the one by @aknrdureegaesr). I think the authors (including @FND) corrected their taggings.

I also added an option to delete unused tags (https://github.com/innoq/iBlog/commit/8d1cfa8e97bb2361d4530acd95ae8b6525bd2640#diff-b7a2501abd8fd84f85c960ca6c9885f6R5). So maybe the simplest solution is that @aknrdureegaesr correct his article with the space-delimited-tag (Entry-ID: 1005279).

[1] `SELECT t.* FROM tags t WHERE t.name LIKE '% %';`
[2] Tags with Entry-reference:
```
SELECT 
	t.*, 
	e.id as 'EntryID',
	e.title as 'EntryTitle'
FROM
	tags t
JOIN taggings ta ON t.id = ta.tag_id
JOIN entries e ON e.id = ta.taggable_id
WHERE
	t.name LIKE '% %'
```
